axoupdater/src/receipt.rs

```diff
@@ -20,6 +20,17 @@ pub struct InstallReceipt {
     pub source: ReleaseSource,
     /// Installed version
     pub version: String,
+    /// Information about the tool used to produce this receipt
+    pub provider: ReceiptProvider,
```

Comment:
we've *always* been producing this?

Reply:
Since the very first commit that introduced receipts. I'd realized we might need it someday, so it was there until we did.